mobile/cmp/menu/impl/Menu.js

```diff
@@ -41,24 +40,18 @@ export const [Menu, menu] = hoistCmp.withFactory({
         if (isEmpty(items)) return null;
         throwIf(!isFunction(onDismiss), 'Menu requires an `onDismiss` callback function');
 
-        return fragment(
-            mask({
-                isDisplayed: true,
-                onClick: () => onDismiss()
-            }),
-            vbox({
-                ref,
-                className,
-                style: {
-                    ...style,
-                    ...layoutProps
-                },
-                item: vbox({
-                    className: 'xh-menu__list',
-                    items
-                })
+        return vbox({
+            ref,
+            className,
+            style: {
+                ...style,
+                ...layoutProps
```

Comment:
Small thing, but any reason to not let the outer `vbox` accept these props directly at the top level and do this itself?  I think you could then destructure off only `menuItem` and `onDismiss` and pass the rest through.